### PR TITLE
istio-csr: Use new test target names (part of migration to makefile-modules)

### PR DESCRIPTION
**Pull request by inteon**
Once we merge this PR, the tests on https://github.com/cert-manager/istio-csr/pull/221 should start succeeding and it should be ready to be merged.

### CODE REVIEW

File: config/jobs/cert-manager/istio-csr/cert-manager-istio-csr-presubmits.yaml
Comment (SgtCoDFish) [inline config/jobs/cert-manager/istio-csr/cert-manager-istio-csr-presubmits.yaml:49]: comment: It feels like if these presets are going to be common for all things using makefile-modules, then it might be worth having something like `preset-makefile-modules` in the future. Or maybe not. Certainly not a blocker for this though!

**Review by SgtCoDFish — APPROVED** [review]
/lgtm
/approve
/hold

The change in base image here makes me slightly nervous that this might break existing tests when this merges (although I've not tested that). I don't think that's a huge problem because I suspect we can move quickly once this merges and complete the shift to makefile-modules.

I've added a hold for that reason - so we can coordinate when all the PRs merge.

**Comment by inteon** [comment]
I'll unhold the PR, since the https://github.com/cert-manager/istio-csr/pull/221 PR is only blocked on this (all tests are passing locally).
Once that PR gets merged, all tests will be working again.
/unhold

**Pull request merged**
